genome_designer/genome_finish/find_insertion_neighborhood.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+from optparse import OptionParser
+import pickle
+import sys
+
+from sam_parse import calcQueryPosFromCigar
+from sam_parse import extractCigarOps
+from sam_parse import SAM
+
+
+def find_insertion_neighborhood(inFile):
```

Comment:
I'm going to trust this works. I'd like to see more testing and demos before we flip the flag on.